modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/NativeFuncs.cs

```diff
@@ -153,7 +153,7 @@ public static partial void
         internal static partial void godotsharp_callable_call_deferred(in godot_callable p_callable,
             godot_variant** p_args, int p_arg_count);
 
-        // GDNative functions
+        // GDExtension functions
 
         // gdnative.h
```

Comment:
This probably warrants further cleanup by @godotengine/dotnet - for this PR I'd suggest leaving it unchanged (same with `modules/mono/glue/runtime_interop.cpp`) so it stays clear that it's an outdated comment and that this API requires a review.

Reply:
OK that make sense, I reverted the rename in comments there.